atc/atccmd/command.go

```diff
@@ -712,20 +744,37 @@ func (cmd *RunCommand) constructAPIMembers(
 				externalHost:  cmd.ExternalURL.URL.Host,
 				baseHandler:   authHandler,
 			},
+			tlsRedirectHandler{
```

Comment:
nit: cmd.ExternalURL.URL.Hostname() & cmd.ExternalURL.URL.Host can be pulled into vars to reduce visual noise since they are repeated a bunch of times